### PR TITLE
Use ov.convert_model

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

Use `openvino.convert_model` instead of `openvino.tools.mo.convert_model`

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
nightly/job/TriggerBetta/552
manual/job/post_training_quantization/245/
manual/job/post_training_quantization/246/

File: tests/openvino/native/test_bias_correction.py [hunk 1]
```diff
@@ -46,7 +45,7 @@ def get_backend() -> OVBiasCorrectionAlgoBackend:
     def backend_specific_model(model: torch.nn.Module, tmp_dir: str):
         onnx_path = f"{tmp_dir}/model.onnx"
         torch.onnx.export(model, torch.rand(model.INPUT_SIZE), onnx_path, opset_version=13, input_names=["input.1"])
```
Comment (KodiaqQ) [inline tests/openvino/native/test_bias_correction.py:47]: Why do we need onnx export still?

Comment (AlexanderDokuchaev) [reply tests/openvino/native/test_bias_correction.py:47]: To keep names of layers

**Comment by KodiaqQ** [comment]
> nightly/job/TriggerBetta/552 manual/job/post_training_quantization/245/ manual/job/post_training_quantization/246/

Was it tested with `openvino-nightly`?

**Comment by AlexanderDokuchaev** [comment]
> > nightly/job/TriggerBetta/552 manual/job/post_training_quantization/245/ manual/job/post_training_quantization/246/
> 
> Was it tested with `openvino-nightly`?

No, only with openvino==2023.2

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**